source/base/trilinos_utilities.cc

```diff
@@ -169,6 +169,25 @@ namespace Utilities
     }
   } // namespace Trilinos
 #endif
+
+
+#ifdef DEAL_II_TRILINOS_WITH_TPETRA
+  namespace Trilinos
```

Comment:
You need to add doxygen documentation for this `Trilinos` namespace to get documentation for `make_rcp`, see https://stackoverflow.com/a/35068555.

Reply:
This does not resolve that problem. 
The Doxygen settings from deal.II seem to suppress the generation of Doxygen entries for aliases.

I just pushed a fix for that problem. If that fix is okay, I will squash the commits.